base/dcp_client_test.go

```diff
@@ -385,3 +385,26 @@ func TestResumeStoppedFeed(t *testing.T) {
 	}
 
 }
+
+// TestBadAgentPriority makes sure we can not specify agent priority as high
+func TestBadAgentPriority(t *testing.T) {
+	if UnitTestUrlIsWalrus() {
+		t.Skip("This test only works against Couchbase Server, since DCPClient requires collections")
+	}
+
+	bucket := GetTestBucket(t)
+	defer bucket.Close()
+
+	feedID := "fakeID"
+	panicCallback := func(event sgbucket.FeedEvent) bool {
+		panic("Should not hit this callback")
```

Comment:
if we panic we'd bail out of the entire test suite - rather than failing this one test.

```suggestion
		t.Errorf("Should not hit this callback")
```

Reply:
done